.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,4 +5,5 @@ Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor G
 - [ ] Is the subject and message clear and concise?
 - [ ] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
 - [ ] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?
-- [ ] Are any related Issues or existing PRs [properly referenced](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) via a Closes (Issue) or Resolved (PR) line?
\ No newline at end of file
+- [ ] Are any related Issues or existing PRs [properly referenced](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) via a Closes (Issue) or Resolved (PR) line?
+- [ ] Are all passwords or private data gathered by this PR obfuscated?
```

Comment:
I know what obfuscate means :). maybe slight improvement for simple english. And maybe point to implementing `postproc()` and maybe a link to an example?

Reply:
> I know what obfuscate means :). maybe slight improvement for simple english.

Thank you! I always doubt since English is not my first language. 

 >And maybe point to implementing `postproc()` and maybe a link to an example?
This is a fantastic idea. I can create an internal page in https://github.com/sosreport/sos/wiki/Data-privacy-and-obfuscation.rst and point the check to that url. What do you think?

Reply:
> Thank you! I always doubt since English is not my first language.

It's my first language, but still had to look it up the first time :smile: 

> This is a fantastic idea. I can create an internal page in https://github.com/sosreport/sos/wiki/Data-privacy-and-obfuscation.rst and point the check to that url. What do you think?

yeah, like the idea

Reply:
I don't think I have access to edit the Wiki, so somebody else will have to do it.
Anyway, do you think it may be better as a section in the 'how to write a plugin' page instead of a completely new page?

Reply:
I don't have access either, we're both in the same stance.

Looking a the Contribution Guidelines, I do see the 2 comments below. So, I think if we could expand on it would make more sense. Maybe others might have other ideas

> * All regex substitutions (i.e. postproc() obfuscations) must include an example in comments
>   * Required as of 4.1. Older regex substitutions may not have examples, but these should be updated over time.

Reply:
I added there https://github.com/sosreport/sos/wiki/How-to-Write-a-Plugin#how-to-prevent-collecting-passwords .

Reply:
OK, I pushed the change with the link to the new section in the wiki, thank you @pmoravec for creating it!